content/slides/data-inspection.qmd

```diff
@@ -18,35 +18,249 @@ the same datum using e.g. `gdalwarp`, `ogr2ogr`.
 :::
 
 
-## Layer Properties window
+## QGIS Layer Properties
 
-...
+![CRS of a QGreenland layer](/_media/qgreenland_layer_properties.png)
+
+::: {.notes}
+In QGIS, right click a layer -> "Properties"
+
+Here you can find abstract, file format ("Storage"), CRS, spatial extent, distance
+units, feature count, geometry type(s), grid width & height, data type, fill value,
+grid pixel size, grid origin, and more.
+:::
 
 
 # Raster data inspection tools
 
-## Raster data inspection with GDAL
+## Raster data inspection: `gdalinfo`
+
+One of the most commonly-available tools, and is used under-the-hood by many other
+tools.
+
+* <https://gdal.org/programs/gdalinfo.html>
+* `gdalinfo --help`
+* `man gdalinfo`
+
+
+## Raster data inspection: `gdalinfo`
+
+_TODO: Should we use a small dataset that we can include in the Git repo, and actually
+run these operations at Quarto render-time?_
+
+```{.text code-line-numbers="1|2-4|5|6,7|45|48|49-87|97|100|101|102|103-115"}
```

Comment:
This magic enables walking through specific lines incrementally. `|` separates increments, `-` notates a range of lines to highlight, and `,` notates a list of lines to highlight.